abot/bot.py

```diff
@@ -36,6 +36,8 @@ def whoami(self) -> Optional['Entity']:
         raise NotImplementedError()
 
     def is_mentioned(self, message_event: 'MessageEvent') -> Optional[str]:
+        if not self.whoami():
```

Comment:
Instead of doing two calls to `whoami()` please store the result in a variable and then check on that. We will probably be able to avoid this in the future with https://www.python.org/dev/peps/pep-0572/, but for now...